website/content/en/blog/clusters-for-all-cloud-tenants.md

```diff
@@ -1,7 +1,12 @@
 ---
 title:  "Clusters for all cloud tenants"
 date:   2022-06-02 13:04:00 +0200
-author: Josh Gavant 
+author: Josh Gavant
+categories:
+- Community
+tags:
+- WG Multi-tenancy
+- Technology
```

Comment:
I think you're getting at the fact that the article is technical and not some sort of announcement. Perhaps "technical" or "deep dive" would fit better?